tests/suites/test_suite_oid.function

```diff
@@ -10,7 +10,7 @@
  * END_DEPENDENCIES
  */
 
-/* BEGIN_CASE depends_on:MBEDTLS_ASN1_WRITE_C*/
+/* BEGIN_CASE */
```

Comment:
Doesn't this need ASN1_WRITE_C to have `mbedtls_asn1_buf`? If not, then it should probably be removed in a different commit than the one that adds MBEDTLS_OID_CERTIFICATE_POLICIES

Reply:
`mbedtls_asn1_buf` is defined in `asn.h` without any dependency.
The `depends_on` was a residue from earlier attempt in the test to use the asn1_write functionality, but there wasn't a need for it eventually.
I'll do it in a different commit